lib/session.ts

```diff
@@ -85,12 +106,12 @@ export async function findSession(req) {
         language,
         country,
       });
-    } catch (e) {
+    } catch (e: any) {
       if (!e.message.toLowerCase().includes('unique constraint')) {
         throw e;
       }
     }
   }
 
-  return session;
+  return { session, website };
```

Comment:
We should not overload this method. For our purpose it's cached anyways. Better to keep the logic clear.

Reply:
Then we have to make a 2nd call to the cached website, outside of this method then.  It's not a big most of the time, but during high traffic scenarios, we should be squeezing as much performance/efficiency as we can out of the collect.